packages/playwright/src/runner/failureTracker.ts

```diff
@@ -40,7 +42,9 @@ export class FailureTracker {
   }
 
   hasReachedMaxFailures() {
-    const maxFailures = this._config.config.maxFailures;
+    let maxFailures = this._config.config.maxFailures;
```

Comment:
There is no point in recomputing it over and over, easier to resolve in constructor once.